src/NUnitTestExtractor/NUnitTestExtractorApp.cs

```diff
@@ -105,213 +86,216 @@ private static void WriteTestDllAndName(TextWriter writer, string dll, string da
         /// <param name="level">Specifies the level of granuality to use for the output: either namepspace, class or function</param>
         /// <param name="writer">The StreamWriter which will be passed to WriteTestDllAndName() </param>
```

Comment:
[Must] You removed the write parameter, so you should remove it from the function documentation too.

Reply:
fixed